server/ai_mediaserver.go

```diff
@@ -72,6 +72,7 @@ func startAIMediaServer(ls *LivepeerServer) error {
 	ls.HTTPMux.Handle("/audio-to-text", oapiReqValidator(handle(ls, multipartDecoder[worker.GenAudioToTextMultipartRequestBody], processAudioToText)))
 	ls.HTTPMux.Handle("/llm", oapiReqValidator(ls.LLM()))
 	ls.HTTPMux.Handle("/segment-anything-2", oapiReqValidator(handle(ls, multipartDecoder[worker.GenSegmentAnything2MultipartRequestBody], processSegmentAnything2)))
+	ls.HTTPMux.Handle("/live-video-start", ls.StartLiveVideo())
```

Comment:
```suggestion
         // This is called by Media Server when stream is ready
	ls.HTTPMux.Handle("/live-video-start", ls.StartLiveVideo())
```

1. I'd add the comment that this endpoint is actually called by Media Server
2. Maybe rename to match `/live-video-to-video

Reply:
Comment added in 9e461a7d2ee59b1cd5603382d403c082b71469ad

@victorges [suggested](https://discord.com/channels/423160867534929930/1296851581240610818) using a `/live/` prefix for these paths which sounds good to me so I did that here. Named it `/live/video-to-video/start` since I suspect we will want to append other things to the route, like maybe `/live/video-to-video/api` or thereabouts